src/quo2/components/tags/status_tags.cljs

```diff
@@ -45,49 +45,74 @@
          [icon/icon
           icon
           {:no-color true
-           :size     12}]]
+           :size     (if (= :large size) 20 12)}]]
         [text/text
          {:size   paragraph-size
           :weight :medium
           :style  {:padding-left 5
                    :color        text-color}} label]]])))
 
 (defn- positive
-  [size theme label]
+  [size theme label blur?]
   [base-tag
    {:size             size
-    :icon             :verified
+    :icon             :i/positive-state
     :background-color colors/success-50-opa-10
     :border-color     colors/success-50-opa-20
     :label            label
-    :text-color       (if (= theme :light)
-                        colors/success-50
-                        colors/success-60)}])
+    :text-color       (cond
+                        ;; The positive tag uses the same color for light themed and dark blur variant
+                        (or (= theme :light) blur?) colors/success-50
+                        (= theme :dark)             colors/success-60)}])
 
 (defn- negative
-  [size theme label]
+  [size theme label blur?]
   [base-tag
    {:size             size
-    :icon             :untrustworthy
+    :icon             :i/negative-state
     :background-color colors/danger-50-opa-10
     :border-color     colors/danger-50-opa-20
     :label            label
-    :text-color       (if (= theme :light)
-                        colors/danger-50
-                        colors/danger-60)}])
+    :text-color       (cond
+                        (= theme :light)           colors/danger-50
+                        ;; The negative tag uses the same color for dark themed and dark blur variant
+                        (or (= theme :dark) blur?) colors/danger-60)}])
+
+(defn- get-color-or-icon-name
+  [{:keys [blur? theme dark-blur-variant light-variant dark-variant]}]
+  (if blur?
+    ;; status tag support only dark blur variant
+    dark-blur-variant
+    (if (= theme :light) light-variant dark-variant)))
 
 (defn- pending
-  [size _ label]
+  [size theme label blur?]
   [base-tag
    {:size             size
-    :icon             :pending
     :label            label
-    :background-color colors/white-opa-5
-    :border-color     colors/white-opa-5
-    :text-color       colors/white-opa-70}])
+    :icon             (get-color-or-icon-name {:blur?             blur?
```

Comment:
`get-color-or-icon-name` could you elaborate?

Reply:
The color and icon (name) change depending on the theme (`light` or `dark`) and blur (`dark blur`). I added a private method to return the data (color or icon) depending on the theme and blur to prevent the repetition of conditional checks. If you felt the method name is confusing, I'm open to suggestions.

Reply:
i just wondering how icon can be a color? oh ok, yeah its confusing for sure, i believe for colors we have already something like themed-color-blur, for icons it could be just condition without any methods

Reply:
I was confused too. Without the custom function it seems easier to grasp, at least to me. The secret is to pass the third argument to override the theme, this way we can always avoid the check `(if (= theme :light) ...)`

```clojure
(if blur?
  colors/white-opa-5
  (colors/theme-colors colors/neutral-10 colors/neutral-80-opa-40 theme))

;; The above should be the same as:
(get-color-or-icon-name {:blur?             blur?
                         :theme             theme
                         :dark-blur-variant colors/white-opa-5
                         :light-variant     colors/neutral-10
                         :dark-variant      colors/neutral-80-opa-40})
```